src/coreclr/pal/src/map/virtual.cpp

```diff
@@ -2140,7 +2141,15 @@ void ExecutableMemoryAllocator::TryReserveInitialMemory()
     int32_t preferredStartAddressIncrement;
     UINT_PTR preferredStartAddress;
     UINT_PTR coreclrLoadAddress;
-    const int32_t MemoryProbingIncrement = 128 * 1024 * 1024;
+
+    // If we manage to reserve the initial memory close to coreclr we might get a better performance
+    // but it's better to turn it off when we run benchmarks for more stable results (always reserve far from coreclr)
```

Comment:
It does not sound right for the benchmarks to measure something else than what customers see.

Reply:
The problem that currently on ARM we have quite shaky results and it's difficult to measure improvements from various changes, e.g:
![image](https://user-images.githubusercontent.com/523221/173536158-2f62e7a0-af6a-4f78-ba23-4f31d40e5aae.png)

I'll experiment on crank what my flags do to a series of measurements and how big RPS is when we're lucky to reserve a piece next to coreclr (and how often) - according to my previous measurements it's 5% larger in that case